docs/src/introduction/overview.md

```diff
@@ -169,6 +171,9 @@ This starts a Fuel node with a volatile database that will be cleared when shut
 
 ### Deploy `counter_contract` To Your Local Fuel Node
 
+> **Note**
+> If you want to deploy your contract to the testnet instead of to a local Fuel node, check out the [Developer Quickstart](https://fuellabs.github.io/fuel-docs/master/developer-quickstart.html).
```

Comment:
```suggestion
> If you want to deploy your contract to the testnet instead of to a local Fuel node, check out the [Developer Quickstart Guide](https://fuellabs.github.io/fuel-docs/master/developer-quickstart.html).
```